workspace.json

```
@@ -780,5 +780,6 @@
         }
       }
     }
-  }
+  },
+  "defaultProject": "examples-angular-nrwl-airlines-booking-desktop-app"
```

Angular is the most often used project. It's the default one we should use generators in. Its the default one we should build.
```suggestion
  "defaultProject": "angular"
```

Instead, change the `start` script in `package.json` to:

```
    "start": "nx serve examples-angular-nrwl-airlines-booking-desktop-app",
```